assets/javascripts/discourse/components/fields/da-trust-levels-field.js.es6

```diff
@@ -0,0 +1,6 @@
+import BaseField from "./da-base-field";
+import Site from "discourse/models/site";
+
+export default BaseField.extend({
+  allTrustLevel: Site.currentProp("trustLevels"),
```

Comment:
It's probably okay because trust levels don't change but ideally you would not use the global `Site` here.